frontend/src/Components/Footer.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+
+const Footer = () => {
+  return (
+    <footer className="footer" id="footer">
+      <span className="col">
+        <h3 className="col-title">Info</h3>
+        <nav className="col-list">
+          <ul>
+            <li>
+              <a href="/">Formats</a>
+            </li>
+            <li>
+              <a href="/">Compression</a>
+            </li>
+            <li>
+              <a href="/">Pricing</a>
+            </li>
+            <li>
+              <a href="/">FAQ</a>
+            </li>
+            <li>
+              <a href="/">Status</a>
```

Comment:
suggestion - I think `<a></a>` tag isn't required in react. You can use `<Link>` for navigating inside app & for external links you can use `<a>`